src/secp256k1/src/cpp/Scalar.cpp

```diff
@@ -281,19 +281,14 @@ std::string Scalar::GetHex() const {
     return ss.str();
 }
 
-void Scalar::SetHex(const std::string& str) const {
+void Scalar::SetHex(const std::string& str) {
```

Comment:
Better to check if `str` have the correct length.

Reply:
fixed